### PR TITLE
Add Magisk module update mechanism

### PR DESCRIPTION
**Pull request by tonytkachenko**

### CODE REVIEW

File: module/module.prop [hunk 1]
```diff
@@ -4,3 +4,4 @@ version=v1.2
 versionCode=3
 author=AdGuard
 description=Copies AdGuard's CA certificate from the user certificate store to the system store and forces Zygisk unmount procedures for certain browsers.
+updateJson=https://github.com/AdguardTeam/adguardcert/raw/master/update.json
```
Comment (tonytkachenko) [inline module/module.prop:7]:
```suggestion
updateJson=https://raw.githubusercontent.com/AdguardTeam/adguardcert/master/update.json
```

**Review by tonytkachenko — COMMENTED** [review]
Changed updateUrl

**Comment by sfionov** [comment]
Hello! Thank you for your contribution!

We'll review it ASAP

File: update.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "version": "v1.2",
+  "versionCode": 3,
+  "zipUrl": "https://github.com/AdguardTeam/adguardcert/releases/download/v1.2/adguardcert-v1.2.zip",
```
Comment (ngorskikh) [inline update.json:4]: Should point to `latest`, probably.

Comment (sfionov) [reply update.json:4]: It is better to have exact version in manifest

**Review by sfionov — APPROVED** [review]

**Pull request merged**